code/modules/reagents/chemistry/holder.dm

```diff
@@ -1112,9 +1112,9 @@
 				if(percent < minimum_percent)
 					continue
 				var/intensity_desc = "a hint of"
-				if(percent > minimum_percent * 2 || percent == 100)
+				if(ISINRANGE(percent, minimum_percent * 2, minimum_percent * 3)|| percent == 100)
 					intensity_desc = ""
-				else if(percent > minimum_percent * 3)
+				else if(percent > minimum_percent * 3)  // CITCHANGE - food now tasty
```

Comment:
you don't actually need this bit anymore, modular man bad

Reply:
The comment?